release_images.yml

```diff
@@ -7,28 +7,20 @@ release_images:
       device_types: ["cpu", "gpu"]
       python_versions: ["py36"]
       os_version: "ubuntu16.04"
-      cuda_version: "cu101"
+      cuda_version: "cu110"
```

Comment:
Please remove "cpu" from `device_types`, or make it a separate entry, because this will apply `disable_sm_tag` to both.